packages/tools/src/utilities/annotationFrameRange.ts

```diff
@@ -47,32 +48,88 @@ export default class AnnotationFrameRange {
     );
   }
 
+  public static setStartRange(
```

Comment:
There seems to be duplicated logic in this class, since lots is implemented in frameRangeUtils now